packages/IMAPClient-Tests.package/ICMockServer.class/README.md

```diff
@@ -0,0 +1 @@
+This is part of the imap server communication mock up.
```

Comment:
I think the term *mock server for testing purposes* would fit better here.

Reply:
I added "offline":
"Mock server for offline testing purposes"